### PR TITLE
FIX: language is reset in metadata section of the template when making changes in parent content profile. [SDCP-265]

### PR DESCRIPTION
**Pull request by GyanP**
We keep updating the template with the new schema without checking if there is any metadata value present before.

### CODE REVIEW

**Comment by GyanP** [comment]
> can you write a test for this pls?

Ya sure.

File: apps/content_types/content_types.py [hunk 1]
```diff
@@ -180,6 +180,8 @@ def _update_template_fields(self, updates, original):
         Finds the templates that are referencing the given
         content profile an clears the disabled fields
         """
+        template_metadata_fields = ['language', 'usageterms']
```
Comment (petrjasek) [inline apps/content_types/content_types.py:183]: why is it just those 2 fields which are affected?

Comment (GyanP) [reply apps/content_types/content_types.py:183]: Because these are the only fields of templates that don't depend on the schema.

Comment (petrjasek) [reply apps/content_types/content_types.py:183]: can you add some comment there regarding it? just to make it clear in the code.

Comment (GyanP) [reply apps/content_types/content_types.py:183]: Sure, I'll.

**Pull request merged**